packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2524,7 +2524,7 @@ async function handler(
         ...evt,
         ...{ metadata: metadataFromEvent, eventType: { slug: eventType.slug } },
       },
-      isNotConfirmed: evt.requiresConfirmation || false,
+      isNotConfirmed: !isConfirmedByDefault,
```

Comment:
also includes bookings with payments not just event types that require confirmation